examples/update_flow_control/update_flow_control.py

```diff
@@ -0,0 +1,254 @@
+# Copyright 2021 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+"""example application to control the Update flow of the Mender client
+"""
+
+import os
+import sys
+import threading
+from datetime import datetime, timedelta
+from queue import Empty, SimpleQueue
+from typing import Any, Tuple
+
+from gi.repository.GLib import Error as DBusError  # type: ignore
+from pydbus import SystemBus  # type: ignore
+
+UPDATE_CONTROL_MAP_PAUSE_ALL = """{
+	"priority": -1,
+	"states": {
+		"ArtifactInstall_Enter": {
+			"action": "pause"
+		},
+		"ArtifactReboot_Enter": {
+			"action": "pause"
+		},
+		"ArtifactCommit_Enter": {
+			"action": "pause"
+		}
+	},
+	"id": "01234567-89ab-cdef-0123-456789abcdef"
```

Comment:
Should we use a random `UUID`?

Reply:
I think this is fine. See also comment about `priority`.

Reply:
Hmm, from the example perspective I would not consider it necessary, but if we are to promote proper usage, maybe we should? 
(sry, answering a question with a question :p)